overlord/ifacestate/helpers_test.go

```diff
@@ -213,3 +214,41 @@ func (s *helpersSuite) TestHotplugSlotInfo(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(slots, DeepEquals, defs)
 }
+
+func (s *helpersSuite) TestFindConnsForHotplugKey(c *C) {
+	st := s.st
+	st.Lock()
+	defer st.Unlock()
+
+	st.Set("conns", map[string]interface{}{
```

Comment:
maybe a comment about how you're setting it on state and then getting it just to avoid having to know the internals `connState`?